DCA632FFFEB8FFB6.json

```diff
@@ -0,0 +1,19 @@
+
+{
+  "gateway_conf": {
+    "gateway_ID": "DCA632FFFEB8FFB6",
+    "servers": [
+      {
+        "server_address": "eu1.cloud.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 0,
```

Comment:
please use a valid, even if approximate, lat/long

Reply:
i've added, sorry for the oversight

Reply:
thx!